src/service.ts

```diff
@@ -331,6 +332,42 @@ export class Service {
     }
     return uri;
   }
+
+  static downloadProject(project: Project, uri?: string) {
+    const zipFile: JSZip = new JSZip();
+    let zipName: string = "download.zip";
+    if (!isUndefined(uri)) {
+      zipName = `${uri}.zip`;
+    }
+    function defineZip(file: File, folder?: any) {
```

Comment:
Is there no type available for the `folder` parameter?

Reply:
It is casted as `object`. But if defined as an object, it shows up this:
`Property 'file' does not exist on type 'object'` (here property `file` means the .file func used for adding files). So I better thought to leave it as `any`.